gemma.cc

```diff
@@ -134,10 +137,11 @@ hwy::AlignedUniquePtr<Weights<TConfig>> LoadWeights(const Path& checkpoint) {
                      sizeof(layer_view->pre_attention_norm_scale), 1, fptr);
     ok &= 1 == fread(&layer_view->pre_ffw_norm_scale,
                      sizeof(layer_view->pre_ffw_norm_scale), 1, fptr);
+    total_size += sizeof(*layer_view);
   }
   if (!ok) {
-    HWY_ABORT("Failed to read from %s - might be a directory, or too small?",
-              checkpoint.path.c_str());
+    HWY_ABORT("Failed to read from %s - might be a directory, or too small? "
+              "expected size: %zu", checkpoint.path.c_str(), total_size);
```

Comment:
When the var is u64, maybe >> 10, then %d and static_cast<unsigned>() to avoid the ugly PRIu64?